package.json

```diff
@@ -190,7 +190,7 @@
     "core-js": "^3.6.5",
     "deep-freeze-strict": "^1.1.1",
     "del": "^6.1.1",
-    "dns-sync": "^0.2.1",
+    "dns-sync": "npm:@amoo-miki/dns-sync@^0.2.1",
```

Comment:
@AMoo-Miki I have been thinking why not publishing under `@opensearch-project`? Or do we have the plan to move these forked packages to `@opensearch-project`?

Reply:
While we could, publishing them under the project name would create an expectation of ownership and support which we don't have the bandwidth for.

Reply:
the only concern is it might not be scale to put on personal repo just for one fix. but it should be ok if we don't have other option.